contrib/ec2/gen-json.py

```diff
@@ -60,13 +65,20 @@
   RemainAfterExit=yes
   ExecStart=/usr/bin/chown -R etcd:etcd /media/etcd
 '''
+ETCD_DROPIN = '''
```

Comment:
No, but since we need etcd to start after the mount, it's better to be explicit than rely on ordering and *hope* things start up in the order we want. Now etcd cannot start until the mount is ready.